docs/source/user_guide/data_check_actions.ipynb

```diff
@@ -99,9 +99,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# let's copy the datetime at row 1 for future use\n",
```

Comment:
`DateTimeFormatDataCheck` isn't run for a binary problem type so all datetime error references are obsolete